cql2/standard/requirements/accent-case-insensitive-comparison/REQ_casei-builtin-function.adoc

```diff
@@ -0,0 +1,9 @@
+[[req_accent-case-insensitive-comparison_casei-builtin-function]]
+[width="90%",cols="2,6a"]
+|===
+^|*Requirement {counter:req-id}* |*/req/accent-case-insensitive-comparison/casei-builtin-function*
+^|A |The server SHALL support a built-in function named `CASEI`.
+^|B |The function SHALL accept one argument that can be a character string literal, the name of a property that evaluates to a string literal or a function that returns a string literal (see rules `characterLiteral`, `propertyName`, `function`).
+^|C |The function SHALL return a character string literal.
+^|D |The function SHALL implement the https://www.w3.org/TR/charmod-norm/#definitionCaseFolding[full case folding] algorithm defined in the implementation guidelines of https://www.unicode.org/versions/Unicode14.0.0[The Unicode Standard, Version 14.0] (see https://www.unicode.org/versions/Unicode14.0.0/ch05.pdf[clause 5.18 Case Mappings, sub-clause Caseless Matching]).
```

Comment:
Consider adding the term “Unicode case folding” to the terms & definitions:

**Unicode case folding**
case folding
process of making two texts which differ only in case identical for comparison purposes

Note: Case folding is meant for the purpose of string matching.

Source: https://www.w3.org/TR/charmod-norm/#definitionCaseFolding

Consider adding and using the term [“Unicode case-insensitive matching”](https://www.w3.org/TR/charmod-norm/#dfn-unicode-case-insensitive) as well.

Reply:
Meeting 2022-01-17: Agreed.